src/Core/Geographic/Coordinates.js

```diff
@@ -224,8 +224,32 @@ export function convertValueToUnit(unitIn, unitOut, value) {
 function Coordinates(crs, ...coordinates) {
     this._values = new Float64Array(3);
     this.set(crs, ...coordinates);
+
+    Object.defineProperty(this, 'surfaceNormal',
+        {
+            configurable: true,
+            get: () => {
+                const result = this._normal || this.computeSurfaceNormal();
+                return result;
+            },
+        });
 }
 
+Coordinates.prototype.computeSurfaceNormal = function computeSurfaceNormal() {
+    // compute a normal only for cartesian coordinate.
+    if (this._internalStorageUnit != UNIT.METER) {
+        throw (new Error('You can only compute cartesian normal when unit is METER'));
+    }
+    // In globe mode (EPSG:4978), we compute the normal.
+    if (this.crs == 'EPSG:4978') {
+        this._normal = ellipsoid.geodeticSurfaceNormal(this);
+        return this._normal;
+    }
+    // In planar mode, normal is the up vector.
+    this._normal = THREE.Object3D.DefaultUp;
```

Comment:
As stated earlier, you must use `new THREE.Vector3(0, 0, 1)` not `DefaultUp`

Reply:
In planar view, DefaultUp is defined as the normal to the plan.
https://github.com/iTowns/itowns/blob/061d19e370e68e22af9e23d82ff4fc9ffc76d28c/src/Core/Prefab/PlanarView.js#L121

It's more generic to use DefaultUp. WDYT ?

Reply:
Nope.
This settings is ignored when tiled are built: gis coordinates are always mapped on x,y so z is always the normal, regardless of DefaultUp.